charts/pega/README.md

```diff
@@ -880,13 +880,13 @@ To support this option,
 3) Update your local Helm repository to the latest version using the command:
     - helm repo update pega https://pegasystems.github.io/pega-helm-charts
 4) Update your values.yaml file to refer to the external secret manager for certificates.
-5) pega supports multiple externalSecrets for custom certificates, add externalSecret names as array of strings to certificatesSecret
+5) pega supports multiple externalSecrets for custom certificates, add externalSecret names as array of strings to certificatesSecrets
```

Comment:
5) Add multiple custom certificates that you maintain as an externally-managed secret, each as a string, separated by a comma in the `certificatesSecrets`  parameter.

Reply:
Fixed this